### PR TITLE
List command for doctor and patient

### LINKED ISSUE #53: Add parser for List command

### PR DESCRIPTION
**Pull request by taniathng**
Resolves #53

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,6 +15,8 @@ src/main/resources/docs/
 /*.log.*
 hs_err_pid[0-9]*.log
 
+/config/settings.json
```
Comment (sam-theman88) [inline .gitignore:18]: I think the file path should be .vscode/settings.json?

File: .vscode/settings.json
Comment (sam-theman88) [inline .vscode/settings.json:1]: I don't think this will affect the app/product but safer to delete it I think?

**Review by sam-theman88 — COMMENTED** [review]
LGTM! Just not sure about the vscode files though

File: .gitignore (hunk 1 above)
Comment (MiloTruck) [reply .gitignore:18]: Should just be `.vscode/`

**Review by sam-theman88 — APPROVED** [review]
LGTM!

**Pull request merged**